packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -341,8 +341,16 @@ export function appendChild(
   parentInstance.appendChild(child);
 }
 
+type DOMContainer =
```

Comment:
I can also export this type from `ReactDOM` if that makes more sense.

Reply:
Would make more sense than duplicating to me.